easybuild/easyconfigs/s/strace/strace-5.14-GCCcore-11.2.0.eb

```
@@ -0,0 +1,28 @@
+easyblock = 'ConfigureMake'
+
+name = 'strace'
+version = '5.14'
+
+homepage = 'https://strace.io/'
+description = """
+strace is a diagnostic, debugging and instructional userspace utility for Linux. It is used to monitor and tamper with
+interactions between processes and the Linux kernel, which include system calls, signal deliveries, and changes of
+process state.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+source_urls = ['https://github.com/strace/strace/releases/download/v%(version)s/']
+sources = [SOURCE_TAR_XZ]
+checksums = ['901bee6db5e17debad4530dd9ffb4dc9a96c4a656edbe1c3141b7cb307b11e73']
+
+builddependencies = [
+    ('binutils', '2.37'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/strace-log-merge', 'bin/strace'],
+    'dirs': ['share']
+}
```

Could you add a sanity check command here, e.g. `strace --help`?
```suggestion
sanity_check_paths = {
    'files': ['bin/strace-log-merge', 'bin/strace'],
    'dirs': ['share']
}

sanity_check_commands = ['strace --help']
```

---

Committing this, as it's a minor change/improvement.